From 06db174f2503e0501a5f42231d834d3fc8e74302 Mon Sep 17 00:00:00 2001
From: Russell Toris <rctoris@wpi.edu>
Date: Mon, 20 May 2013 14:32:58 -0400
Subject: [PATCH 1/4] postinst script added

---
 debian/postinst |    4 ++++
 1 file changed, 4 insertions(+)
 create mode 100755 debian/postinst

diff --git a/debian/postinst b/debian/postinst
new file mode 100755
index 0000000..6382446
--- /dev/null
+++ b/debian/postinst
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+sudo setcap cap_net_raw+ep /opt/ros/groovy/lib/youbot_driver/base_arm_gripper_test
+sudo setcap cap_net_raw+ep /opt/ros/groovy/lib/youbot_driver/displayIpAddress
\ No newline at end of file
-- 
1.7.9.5

